index.js

```diff
@@ -117,8 +118,30 @@ const cleanUpLinks = (input) => {
   return input
 }
 
+const normalizeLink = (entityMap, link, removeUnresolved) => {
+  const resolvedLink = getResolvedLink(entityMap, link)
+  if (resolvedLink === UNRESOLVED_LINK) {
+    return removeUnresolved ? resolvedLink : link
+  }
+  return resolvedLink
+}
+
+const maybeNormalizeLink = (maybeLink, entityMap, removeUnresolved) => {
+  if (Array.isArray(maybeLink)) {
+    maybeLink = maybeLink.map((maybeArrayLink) => maybeNormalizeLink(maybeArrayLink, entityMap, removeUnresolved))
+    if (removeUnresolved) {
+      maybeLink = maybeLink.filter((val) => val !== UNRESOLVED_LINK)
+    }
+  } else if (typeof maybeLink === 'object') {
+    if (isLink(maybeLink) || isResourceLink(maybeLink)) {
+      maybeLink = normalizeLink(entityMap, maybeLink, removeUnresolved)
```

Comment:
Are these args correct here?

Reply:
Oh, it looks like normalizeLink and maybeNormalizeLink have the same args in different order.